examples/table/table.md

```diff
@@ -0,0 +1,13 @@
+# Table
+
+The table constructor expression looks like an array constructor. `add` and `put` langlib functions can be used to update a table. `put` method will update the table if there already is an entry with a given key or add a new entry otherwise, while the `add` method will panic if there's already an entry with the given key. The `foreach` statement will iterate over a table's rows in their order. Use `t[k]` to access a row using its key.
```

Comment:
```suggestion
The table constructor expression looks like an array constructor. The `add` and `put` langlib functions can be used to update a table. The `put` method will update the table if there is an entry with a given key or add a new entry otherwise, while the `add` method will panic if there is an entry with the given key. The `foreach` statement will iterate over a table's rows in their order. Use `t[k]` to access a row using its key.
```